src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    public void doWork() {
+        System.out.println("Bulldozer started its work.");
```

Comment:
The message should indicate what type of `Machine` is working right now. Consider including the class name in the message for clarity, e.g., 'Bulldozer is working.'